app/models/cohort_list.rb

```diff
@@ -67,6 +68,7 @@ def parse_rows!
     self.rows =
       data
         .map { |raw_row| raw_row.to_h.transform_keys { _1.downcase.to_sym } }
+        .map { |row| row.transform_values { _1&.sub(/^="(.*)"$/, '\1') } }
```

Comment:
I added automatic unquoting of fields so that users can directly copy-and-paste into the upload sheet. It should be safe to automatically do this on all fields coming in.